docs/local_install_skaffold.md

```diff
@@ -1,51 +1,266 @@
-# Local install of Substra using kubernetes and skaffold
+# Local installation of Substra using Kubernetes and Skaffold
+
+> This is an ongoing document, please feel free to reach us or to raise any issue.
+
+This guide will help you to run the Substra platform on your machine in development mode, with a two nodes setup.
+
+- [Local installation of Substra using Kubernetes and Skaffold](#local-installation-of-substra-using-kubernetes-and-skaffold)
+  - [1. Substra Setup](#1-substra-setup)
+    - [General knowledge](#general-knowledge)
+    - [Hardware requirements](#hardware-requirements)
+    - [Software requirements](#software-requirements)
+      - [1. Kubectl](#1-kubectl)
+      - [2. Minikube](#2-minikube)
+      - [3. Helm](#3-helm)
+      - [4. Skaffold](#4-skaffold)
+    - [Virtualization](#virtualization)
+    - [Get the source code](#get-the-source-code)
+    - [Configuration](#configuration)
+      - [Minikube (Ubuntu only)](#minikube-ubuntu-only)
+      - [Helm init](#helm-init)
+      - [Network](#network)
+  - [3. Running the platform](#3-running-the-platform)
+    - [Start Substra](#start-substra)
+    - [Stop Substra](#stop-substra)
+    - [Reset Substra](#reset-substra)
+  - [4. Login, password and urls](#4-login-password-and-urls)
+    - [Credentials and urls](#credentials-and-urls)
+    - [Browser extension](#browser-extension)
+    - [Substra CLI config & login](#substra-cli-config--login)
+  - [5. Troubleshooting](#5-troubleshooting)
+    - [Virtualization issues](#virtualization-issues)
+    - [Kubectl useful commands](#kubectl-useful-commands)
+    - [Minikube useful commands](#minikube-useful-commands)
+    - [Tiller](#tiller)
+    - [[WIP] Ongoing issues](#wip-ongoing-issues)
+  - [6. Need help?](#6-need-help)
+  - [7. Further resources](#7-further-resources)
+  - [8. Acknowledgements](#8-acknowledgements)
+
+___
+
+When everything is ready, you will be able to start Substra with:
 
-This guide will help you run the Substra platform on your machine in development mode, with a 2 organisations setup.
+```sh
+# Ubuntu only
+minikube start --cpus 6 --memory 8192 --disk-size 50g --kubernetes-version='v1.15.4'
+
+# In 3 different terminal windows, in this order:
+# In the repository hlf-k8s
+skaffold dev
+
+# In the repository susbtra-backend
+skaffold dev
+
+# In the repository substra-frontend
+skaffold dev
+```
+
+## 1. Substra Setup
+
+### General knowledge
+
+I order to install Substra, it is *recommended* to be comfortable with your package manager and to have some basic knowledge of Unix (Mac or Ubuntu/Debian) administration and network.
```

Comment:
I would add Kubernetes basic knowledge as well?

Reply:
You're right, thank you!